torch/fx/experimental/symbolic_shapes.py

```diff
@@ -1548,16 +1548,32 @@ def _update_divisible(self):
 
         self.divisible = new_divisible
 
+    @_lru_cache
+    def _is_divisible(self, x, y):
+        return x % y == 0 or x % y in self.divisible
+
+    @_lru_cache
+    def _elim_floor(self, x, y):
+        if type(x) is sympy.floor and type(x.args[0]) is sympy.Mul:
+            m1, m2 = x.args[0].args
+            ry = 1 / y
+            if m2 == ry and self._is_divisible(m1, y):  # inner mul is div
+                return m1
+        elif type(x) is FloorDiv:
+            if x.divisor == y and self._is_divisible(x.base, y):
+                return x.base
+
     @_lru_cache
     def simplify(self, expr: "sympy.Expr") -> "sympy.Expr":
         expr = self.replace(expr)
-        if expr.has(FloorDiv):
+        if expr.has(FloorDiv) or expr.has(sympy.floor):
```

Comment:
Need to use `(expr.has(FloorDiv) or expr.has(sympy.floor)) and expr.has(sympy.Mul)` here